app/views/phases/_submissions_stats.html.erb

```diff
@@ -15,7 +15,7 @@
                 Total Submissions
                 </h3>
                 <p class="usa-summary-box__text text-primary-darker text-bold">
-                Evaluations due by <%= @phase.evaluation_form.closing_date.strftime('%m/%d/%Y') %>
+                Evaluations due by <%= if @phase.evaluation_form then @phase.evaluation_form.closing_date.strftime('%m/%d/%Y') end %>
```

Comment:
I've rarely seen the `if .. then .. end` conditional form, there are a few different ways to write this that are a little more concise. It should fallback to 'N/A' if there is no form too. imo the [safe-navigation operator](https://ruby-doc.org/core-2.6/doc/syntax/calling_methods_rdoc.html#label-Safe+navigation+operator) works better here..
```suggestion
                Evaluations due by <%= @phase.evaluation_form&.closing_date&.strftime('%m/%d/%Y') || "N/A" %>
```

Reply:
also this should probably be moved to a helper function, to help reuse the logic, since we use the same date a few places.